README.md

```diff
@@ -65,6 +65,17 @@ In addition to running the tests in the converted modules we also have
 * Review the [roadmap](https://github.com/nodejs/abi-stable-node/milestones) and see how you
 can help accelerate this project.
 
+**Badges**
+
+The use of badges is recommended to indicate the minimum version of N-API
+required for the module. This helps to determine which Node.js major versions
+are supported. The following badges are available:
```

Comment:
What do you think to link here the N-API version matrix https://nodejs.org/dist/latest/docs/api/n-api.html#n_api_n_api_version_matrix ?